source/reference/color_schemes.rst

```diff
@@ -0,0 +1,256 @@
+=============
+Color Schemes
+=============
+
+
+Overview
+========
+
+Color schemes define the colors
+used to highlight source code in Sublime Text views
+and the colors use for different elements
+found in the editing area:
+background, foreground, selection, caret, etc.
+
+
+File Format
+===========
+
+Color scheme files use the Property List format
+and have the .tmTheme extension.
+The file names of color scheme files
+are displayed in the **Preferences → Color Scheme** menu.
+
+The file format of color schemes
+is inherited from Textmate.
+
+.. note::
+
+   Sublime Text uses the .tmTheme extension for color scheme files
+   to maintain compatibility with Textmate.
+   Confusingly, Sublime Text also has a notion
+   of a user interface (UI) theme.
+   A UI theme is a set of styles and decorations
+   to alter the look of the editor’s UI.
+   It’s important to remember
+   that UI themes and color schemes
+   are two different customization mechanisms.
+   Generally speaking, it is far less complex
+   to create a new color scheme
+   than to create a new UI theme.
+
+
+Where to Store Color Schemes
+============================
+
+You can keep color scheme files anywhere under Packages
+(even inside directories nested multiple levels deep).
+Most frequently, you will find color schemes
+inside their own directory.
```

Comment:
I would argue that this is false. Since most packages that have Color Schemes as a priority will have them (or mostly only one) in the main package folder.